src/include/daos_fs.h

```diff
@@ -657,7 +658,7 @@ dfs_write(dfs_t *dfs, dfs_obj_t *obj, d_sg_list_t *sgl, daos_off_t off,
  *
  * \param[in]	dfs	Pointer to the mounted file system.
  * \param[in]	obj	Opened file object.
- * \param[in]	iod	IO descriptor of file view.
+ * \param[in]	iod	IO descriptor for list-io (limit of 64 ranges).
```

Comment:
i need to change that comment to 4k. will do it after all tests run with no testing.